### ISSUE TITLE
Interprocedural analysis

### ISSUE THREAD

**Issue by bchurchill** [StanfordPL/stoke#185]
This is low priority since I don't know that it will be extraordinarily helpful, but it may improve analysis precision.  The idea is to implement interprocedural analysis (k-CFA) to get more precision for all of the other analysis.  This would probably help several of our projects, but it's unclear how much.  Depends on #184.
